pandas/_libs/src/parser/tokenizer.c

```diff
@@ -1433,13 +1433,14 @@ PANDAS_INLINE void uppercase(char *p) {
 int to_boolean(const char *item, uint8_t *val) {
     char *tmp;
     int i, status = 0;
-    int bufsize = sizeof(char) * (strlen(item) + 1);
+    size_t length0 = (strlen(item) + 1);
+    int bufsize = length0;
```

Comment:
This doesn't introduce any new warnings right? Implicitly casting size_t to int?

Reply:
checked now, looks ok. anyway the original line's RHS looks like it was size_t as well, no?